scripts/component_generation/main.go

```diff
@@ -565,16 +565,16 @@ func Spreadsheet(srv *sheets.Service, sheetName string, spreadsheet chan struct
 			batchSize--
 			fmt.Println("Batch size: ", batchSize)
 			if batchSize <= 0 {
-				row := &sheets.ValueRange{
-					Values: values,
-				}
-				response2, err := srv.Spreadsheets.Values.Append(spreadsheetID, sheetName, row).ValueInputOption("USER_ENTERED").InsertDataOption("INSERT_ROWS").Context(context.Background()).Do()
-				values = make([][]interface{}, 0)
-				batchSize = 100
-				if err != nil || response2.HTTPStatusCode != 200 {
-					fmt.Println(err)
-					continue
-				}
+				// row := &sheets.ValueRange{
```

Comment:
This is done for testing. Will be removed.

Reply:
Gotcha